tests/phpunit/tests/test-options.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Test options.
+ */
+class Options_Test extends PLL_UnitTestCase {
+
+	public function test_save_all() {
+		$options = self::create_options(
+			array(
+				'default_lang' => 'en',
+				'force_lang'   => 1,
+			)
+		);
+
+		// Test the value in DB.
```

Comment:
You could add messages to your assertion and remove comment if you'd like.

Reply:
Smort.
Done in [7d3b8b2](https://github.com/polylang/polylang/pull/1579/commits/7d3b8b234d2e57605b9a83cdf4ff16e9b07c7032).